Formula/b/bash.rb

```diff
@@ -112,11 +112,32 @@ def install
     # Homebrew's bash instead of /bin/bash.
     ENV.append_to_cflags "-DSSH_SOURCE_BASHRC"
 
+    bash_loadables_path=[
+      "#{lib}/bash",
+      # Stock Bash paths; keep them for backwards compatibility.
+      "/usr/local/lib/bash",
+      "/usr/lib/bash",
+      "/opt/local/lib/bash",
+      "/usr/pkg/lib/bash",
+      "/opt/pkg/lib/bash",
+      ".",
+    ].join(":")
+    ENV.append_to_cflags "-DDEFAULT_LOADABLE_BUILTINS_PATH='\"#{bash_loadables_path}\"'"
```

Comment:
Given that was pass `--prefix` to `configure`, it seems odd for the build to not automatically add this directory to `DEFAULT_LOADABLE_BUILTINS_PATH`. Seems like it could be an upstream bug?

Reply:
Could be. What you said makes sense to me, but I haven't explored that path. I'll ask on their mailing list, since I was planning on asking about something else anyway.